### DISCUSSION TITLE
Orbit issues?

### DISCUSSION THREAD

**Discussion by jonsberndt** [JSBSim-Team/jsbsim#522] [Support requests]
It used to be that the ball_orbit script produced a run that did not deviate from the altitude very much. For some reason this same run now seems to be failing horribly.

**Comment by jonsberndt** [comment]
Disregard this. I updated from the repo and it looks like all is working.

**Comment by seanmcleod** [comment]
@jonsberndt I pulled the latest from the repo and built and I get this output for the altitude.

![image](https://user-images.githubusercontent.com/824856/144514740-b3e4e7e5-3a28-457c-907b-e6c1ba8609b4.png)

Is this is what you were expecting?

**Comment by seanmcleod** [comment]
Ah, I just realised I was running the script `ball.xml` and not `ball_orbit.xml`.

The `ball_orbit` script does report 800,000ft down to 3 decimal places for all reports.

```sh
Time Notify (Event 0) executed at time: 10200.005000
    Height AGL (ft):          = 800000.000105
    Altitude (Geod, ft):      = 800000.000105
    Latitude (Geod, deg):     = -0.000000
    Vehicle Radius (ft):      = 21725646.325565
    Inertial Vel Mag (ft/s):  = 25473.404282
    Body X Velocity (ft/s):   = 19623.958214
    Simulation Frame:         = 2040001.000000
    Density:                  = 0.000000
    Angular momentum (ft^2/s) = 553426172137.420532
    Inclination (deg)         = 0.000000
    Right ascension (deg)     = 0.000000
    Argument of perigee (deg) = 325.231134
    Period (s)                = 5374.952767
    Eccentricity              = 0.001507
    Apoapsis radius (ft)      = 21791206.648235
    Periapsis radius (ft)     = 21725646.325565
    True anomaly (deg)        = 360.000000

End: Friday December 03 2021 00:43:21 (HH:MM:SS)
```

**Reply by jonsberndt** [reply]
> Ah, I just realised I was running the script `ball.xml` and not `ball_orbit.xml`.
> 
> The `ball_orbit` script does report 800,000ft down to 3 decimal places for all reports.

Hi, Sean - thanks. I was using a version of JSBSim from about  year ago and there were apparently some changes in it. I started with a fresh checkout and it looks good.
